src/typeDefs/inputs.ts

```diff
@@ -113,6 +113,42 @@ export const inputs = gql`
     review: String
   }
 
+  input FundInput {
+    name: String!
+    organizationId: ID!
+    taxDeductible: Boolean!
+    defaultFund: Boolean!
+    archived: Boolean!
+  }
+
+  input FundCampaignInput {
+    name: String!
+    parentFundId: ID
+    pledgeId: ID
```

Comment:
Can these be `null`? Should the FundCampaign always be associated with a pledge or fund?